datamodel_code_generator/parser/jsonschema.py

```diff
@@ -548,41 +548,59 @@ def parse_enum(
 
     def parse_ref(self, obj: JsonSchemaObject, path: List[str]) -> None:
         if obj.ref:
-            ref: str = obj.ref
-            # https://swagger.io/docs/specification/using-ref/
-            if obj.ref.startswith('#'):
-                # Local Reference – $ref: '#/definitions/myElement'
-                pass
-            elif '://' in ref:
-                # URL Reference – $ref: 'http://path/to/your/resource' Uses the whole document located on the different server.
-                raise NotImplementedError(f'URL Reference is not supported. $ref:{ref}')
+            reference = self.model_resolver.get_by_path(obj.ref)
+            if not reference or not reference.loaded:
 
-            else:
-                # Remote Reference – $ref: 'document.json' Uses the whole document located on the same server and in
-                # the same location. TODO treat edge case
-                relative_path, object_path = ref.split('#/')
-                full_path = self.base_path / relative_path
-                with full_path.open() as f:
-                    if full_path.suffix.lower() == '.json':
-                        import json
-
-                        ref_body: Dict[str, Any] = json.load(f)
-                    else:
-                        # expect yaml
-                        import yaml
+                # https://swagger.io/docs/specification/using-ref/
+                if obj.ref.startswith('#'):
+                    # Local Reference – $ref: '#/definitions/myElement'
+                    pass
+                else:
+                    relative_path, object_path = obj.ref.split('#/')
+                    if obj.ref.startswith('https://') or obj.ref.startswith('http://'):
```

Comment:
```suggestion
                    if obj.ref.startswith(('https://', 'http://')):
```
iirc startswith supports tuples.

Reply:
Oh, great!!
I don't know the method supports tuples.
Thank you